set.mm

```diff
@@ -428916,9 +428928,13 @@ which can be either a noun or an adjective (furthermore, "analog" has
        the confounding meaning "not digital"); therefore, "analogue" is used
        for the noun and "analogous" for the adjective.  We favor regular
        plurals, e.g., "formulas" instead of "formulae", "lemmas" instead of
-       "lemmata".
+       "lemmata".  We use the serial comma (Oxford comma) in enumerations.  We
+       use commas after "i.e." and "e.g.".
        </p>
        <p>
+       We avoid beginning a sentence with a symbol (for instance, by writing
+       "The function F is ..." instead of "F is...").</p>
```

Comment:
I don't think I've heard of this so I'm not sure about it

The only two examples in my mathbox (and on the website) are:
+ https://us.metamath.org/mpeuni/fltltc.html
+ https://us.metamath.org/mpeuni/fltnlta.html

> (in retrospect, these could both be generalized)

Reply:
This is a common convention for mathematical writing.